src/app/ApplicationTemplate.Business/ForcedUpdates/UpdateRequiredServiceMock.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Reactive.Linq;
+using ApplicationTemplate.DataAccess;
+
+namespace ApplicationTemplate.Business;
+
+/// <summary>
+/// Mock implementation of the <see cref="IUpdateRequiredService"/>.
+/// </summary>
+public sealed class UpdateRequiredServiceMock : IUpdateRequiredService
```

Comment:
Same comment about `IDisposable` if it's applicable.

Reply:
@jeanplevesque  Should I really add IDisposable to mock implementations? I could see maybe adding it to the interface, but even then, I feel like this should not be necessary? Like, if someone wants something extra later, they can just add the disposable interface?

Reply:
Yeah it would be good to add it because you create a disposable. In the context of automated tests, multiple instances of this class may be created and it would make sense to dispose them.